deployments/deployment.yaml

```diff
@@ -1058,4 +1139,8 @@ parameters:
   - name: CLIENTS_PULP_LOG_PARSER_S3_FILE_PREFIX
     description: path prefix to store pulp logs at in s3
   - name: CLIENTS_ROADMAP_SERVER
-    description: URL to the roadmap service (i.e. https://console.stage.redhat.com/api/roadmap/v1)
\ No newline at end of file
+    description: URL to the roadmap service (i.e. https://console.stage.redhat.com/api/roadmap/v1)
+  - name: SUSPEND_SNAPSHOT_CLEANUP
+    description: whether to not run the daily job to delete outdated snapshots
+    required: false
+    value: "false"
```

Comment:
this is unused, and looks like the new job is using `SUSPEND_CRON_JOB` instead. I'm guessing this can just be removed

Reply:
oops :) meant to update that value but yes just using the existing param is better!